py-polars/polars/expr/datetime.py

```diff
@@ -1493,9 +1496,9 @@ def days(self) -> Expr:
         """
         return wrap_expr(self._pyexpr.duration_days())
```

Comment:
I missed this in my initial review, but the Rust side should be renamed as well (both the bindings in py-polars and the Rust expressions/functions.

```suggestion
        return wrap_expr(self._pyexpr.dt_total_days())
```